### DISCUSSION TITLE
AHK

### DISCUSSION THREAD

**Discussion by abdulla-nsour** [AbdullaNsour/AHK_shortcut#5] [Q&A] [answered]
Do you know AHK (autohotkey) ?

**Comment by Zakon-01** [comment, marked as answer]
Yes AHK is the best Scripting language I have used for 3 years ago,  it gives u the power of doing ur job smarter not harder.
AutoHotkey gives you the freedom to automate any desktop task. It's small, fast and runs out-of-the-box. Best of all, it's free & open-source.
